source/community/release_notes/quince.rst

```diff
@@ -27,14 +31,36 @@ Instructor Experiences
 Administrators & Operators
```

Comment:
There are list of operational changes that have been collected in the wiki at https://openedx.atlassian.net/wiki/spaces/COMM/pages/3726802953/Pending+Release+Quince#Operational-%E2%9A%99%EF%B8%8F -- I usually just copy and paste from there. 

If any of these operational changes sound like something that will break an installation, I put them under Breaking Changes instead.